demo_grid/src/Grid/Query/ProductQueryBuilder.php

```diff
@@ -248,12 +239,27 @@ private function getQueryBuilder(array $filterValues): QueryBuilder
                 continue;
             }
 
+            if (version_compare(_PS_VERSION_, '8.0', '<')) {
+                if ('price_tax_excluded' === $filterName) {
+                    if (isset($filter['min_field'])) {
+                        $qb->andWhere('ps.`price` >= :price_min');
+                        $qb->setParameter('price_min', $filter['min_field']);
+                    }
+                    if (isset($filter['max_field'])) {
+                        $qb->andWhere('ps.`price` <= :price_max');
+                        $qb->setParameter('price_max', $filter['max_field']);
+                    }
+
+                }
+            }
+            /*
             if ('category' === $filterName) {
                 $qb->andWhere('cl.`name` LIKE :category');
                 $qb->setParameter('category', '%' . $filter . '%');
 
                 continue;
             }
+            */
```

Comment:
I think we should delete this commented code to approve the PR. Commented code is often commented the rest of his life :D  

https://dev.to/codebyamir/the-risks-of-dead-code-5bg

Reply:
I don't think so, at least in this case - for future improvement. This commented code can be back once `addFilter(category` is added.

Reply:
It sounds reasonable, removed as we have a poll. I read in many PRs that we need to comment an unused code as Deprecated in the next minor version, then remove it in the next major version. By the way, the author of this snippet is Pierre, I'm just the author of two comment signs, I hope he won't be upset if one day he come back here :pray: